Backend/router/v1/user.js

```diff
@@ -26,6 +25,13 @@ router.all('/', (req, res) => {
   res.send("this is user root");
 });
 
+function password_length_check(length){
+  if(length < MIN_PASSWORD_LENGTH) {
```

Comment:
These 4 lines can be combined into one. Can you try?

Reply:
I can do it!